Dockerfile

```diff
@@ -1,38 +1,21 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
+FROM praekeltfoundation/django-bootstrap
 
-RUN pip install --upgrade pip
 
-ENV PROJECT_ROOT /deploy/
-ENV DJANGO_SETTINGS_MODULE tuneme.settings.docker
 
-WORKDIR /deploy/
+RUN apt-get-install.sh gettext
 
-COPY tuneme /deploy/tuneme
-COPY polls /deploy/polls
-COPY locale /deploy/locale
-ADD manage.py /deploy/
-ADD requirements.txt /deploy/
-ADD setup.py /deploy/
-ADD README.rst /deploy/
-ADD VERSION /deploy/
+ENV PROJECT_ROOT=/app/ \
+    DJANGO_SETTINGS_MODULE=tuneme.settings.docker \
+    APP_MODULE="tuneme.wsgi:application"
 
-RUN pip install -e .
-
-RUN mkdir -p /etc/supervisor/conf.d/
-RUN mkdir -p /var/log/supervisor
-RUN rm /etc/nginx/sites-enabled/default
+ENV CELERY_APP=tuneme \
```

Comment:
These seem specific to Celery while the others seem specific to the Django app. So it's mostly a question of style. It's not critical to have the minimum possible number of layers all the time...just good to not have way more than necessary.